README.md

```diff
@@ -1,14 +1,22 @@
 # Brisk Docs
 
-## A cute slogan goes here
+> Tools to document your mono-repos and React Packages
 
-TODO: Explain all this
+Brisk is an Atlassian Initiative to help with documenting code. It contains two main parts:
 
-## Getting running
+## [The Brisk Website Generator](./packages/website)
 
-```
-yarn global add bolt
-bolt
-```
+This is packaged as `@brisk-docs/website` and is designed as an all-inclusive documentation
+tool for mono-repos containing react components and other packages. Check out [its documentation](./packages/website)
+for more information on how to use it.
 
-This should install and link all the `node_modules`
+## Support Packages
+
+As part of being maximally useful, we want to make sure where possible, our helper components exist as
+their own packages. As such, there are a number of other libraries we maintain that you might be interested
+in, even if you don't need the complete set of things that brisk does:
+
+- [file viewer](./packages/file-viewer) - simple package to run a live component example alongside displaying its code
+- [react changelogs]('./packages/react-changelogs) - Component that accepts raw changelog files and formats them for display. Allows filtering to semver versions
+- [extract-react-types](https://extract-react-types.com/) - a way to extract and display types for components, works on both flow and typescript
```

Comment:
Nit: should we link to the github repo for consistency and then have the repo link out to the website? Nice website btw ^^